### DISCUSSION TITLE
[Question][Hw3] Cannot access to KCLOUD VPN today

### DISCUSSION THREAD

**Discussion by gayeun4195** [prosyslab-classroom/cs348-information-security#699] [Q&A]
Name: Gaeun Noh

Hello, I'm trying to access the server using KCLOUD VPN. However, the KCLOUD VPN login seems impossible since this morning.

Is there any other way to access the CS348 server? Thank you for the answer in advance!

**Comment by KAIST-JongchanPark** [comment]
There is an announcement mail from KCLOUD, which you can get a temporary account.

I think You may already receive it via KAIST email.

Please follow that email.

**Comment by KAIST-JongchanPark** [comment]
@m-spitfire Please do not post what contains personal information.

**Comment by m-spitfire** [comment]
Sorry, I didn't notice.
